Dockerfile

```diff
@@ -1,19 +1,32 @@
 FROM rust:latest
-COPY . /app
+MAINTAINER alex@gnosis.pm
 
-RUN apt-get update \
- && apt-get install -y --no-install-recommends \
-    nano \
-    sudo \
-    sftp \
-    lftp  
-
-COPY tasks/cron-task /etc/crontabs/root
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
 
 RUN apt-get update && apt-get -y install cron
+RUN apt-get -y install lftp
+RUN apt-get -y install nano
+
+WORKDIR /app
+
+COPY src/. src/.
+COPY Cargo.toml .
+COPY Cargo.lock .
+RUN cargo build
+
+#support for sftp
+EXPOSE 22
+
+COPY scripts/. scripts/.
+
+#create config file for validation script
+RUN mkdir /app/config
+RUN echo '1' > /app/config/lastestContributionDate.txt
```

Comment:
Are these files somehow persisted beyond a single session? Otherwise we'd start verifying all contributions on every restart wouldn't we?

Reply:
Yeah, the idea is to echo not just one, but an environment variable into these files. Was planned in the next improvement PR. 

Is there a better way to do it? I was also thinking to work only with envs and not use any files to store any values. But then I can some issues with the cron job, as 
```
*  *  *  *  *  source ./app/scripts/validationAndPreparation.sh >> /var/log/cron.log 2>&1

```
was not really working

Reply:
I was able to use ENV variables only and this is no longer needed